docker/server/nginx/etc/nginx/conf.d/universe.marathon.mesos.conf

```diff
@@ -53,6 +53,10 @@ server {
     set $dcos_release_version 1.12;
   }
 
+  if ($http_user_agent ~ ".*dcos\/1\.13.*") {
```

Comment:
This is for local universe and is enforced above in Line 34. 
For prod universe server, this is enforced at https://github.com/mesosphere/prod-universe/blob/master/nginx/etc/nginx/conf.d/universe.mesosphere.com.conf#L44